ui/src/app/views/profile-registration-step-status/component.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+
+import { Button } from 'app/components/buttons';
+import { ClrIcon } from 'app/components/icons';
+import colors from 'app/styles/colors';
+import { reactStyles } from 'app/utils';
+
+const styles = reactStyles({
+  container: {
+    backgroundColor: '#fff',
+    color: '#262262',
+    flex: '0 0 420px',
+    borderRadius: 8,
+    marginBottom: '10px',
+    padding: 21
+  },
+  title: {
+    color: colors.purple[0],
+    width: 420,
+    fontSize: 16,
+    fontWeight: 600,
+    marginBottom: 6
+  },
+  body: {
+    display: 'flex'
+  },
+  buttonContainer: {
+    width: 'calc(50%)',
+    marginTop: 6
+  },
+  detailsContainer: {
+    width: 'calc(50%)',
+    marginLeft: 40
+  },
+  button: {
+    height: 38
+  },
+  successButton: {
+    backgroundColor: '#8BC990',
+    cursor: 'default'
+  }
+});
+
+interface Props {
```

Comment:
nit: would give this a more specific name

Reply:
so this is a stylistic question I was going to ask the team. When we define State/Props interfaces, should we be prefixing the whole name of the class?

I think for State, it's unnecessary to do so since it's internal to the class. For example, we do not define the css variable above as `profileRegistarationStepStatusStyles`.

For Props, there is an argument that it should be named since it is exposed outside of the class but I think we can define that in the export statement like I did below. That said, the only gain here is that we just save on a little bit of typing but I did it this way since I don't see much of a downside.

Reply:
Discussed. We like the consistency between this generic `Props` and `styles` above. Decided to make this the default practice and may refactor other code to be consistent with this.